.env.development

```diff
@@ -23,3 +23,4 @@ ENABLE_NOTICES=''
 # IDP_SLUG=''
 # uncomment the line below to host the site at /{HOSTNAME}/
 # ENABLE_PATH_PREFIX = true
+USE_MOCK_DATA=''
```

Comment:
Why are you using this env variable?

Reply:
was mentioned in the portal configuration I see no use of it however but left it there as this env file is just used during dev. mode.